usersession/client/client.go

```diff
@@ -307,6 +312,59 @@ func (client *Client) ServicesStop(ctx context.Context, services []string) (stop
 	return stopFailures, err
 }
 
+func (client *Client) ServicesRestart(ctx context.Context, services []string) (restartFailures []ServiceFailure, err error) {
+	restartFailures, _, err = client.serviceControlCall(ctx, "restart", services)
+	return restartFailures, err
+}
+
+func (client *Client) ServicesReloadOrRestart(ctx context.Context, services []string) (restartFailures []ServiceFailure, err error) {
+	restartFailures, _, err = client.serviceControlCall(ctx, "reload-or-restart", services)
+	return restartFailures, err
+}
+
+// UserServiceUnitStatus corresponds to the systemd.UnitStatus in order to be able
```

Comment:
mmh, do we need User in the name? most things here don't have it and it's overall under usersession

Reply:
This is a good point, removed the first user part